src/domain/accounts/services/src/authentication_service_impl.rs

```diff
@@ -145,6 +145,35 @@ impl AuthenticationServiceImpl {
             },
         )
     }
+
+    async fn ensure_predefined_accounts_registration(&self) -> Result<(), InternalError> {
```

Comment:
Can we keep the predefined accounts in memory?

This lazy init will bite us in the ass when we run multiple api servers. Also it only creates accounts in DB - if we remove some predefined account we'll need to clean up DB manually.

Reply:
(suggested in a different commend to make predef accounts a concern of Login+Password provider)

Reply:
They must be stored in database. A predefined account may launch a flow, and their ID must be stored as a foreign key in flow data structures. Soon we would allow access keys, those also have to be associated with an account record in the database. Relaxing database-level integrity (i.e. store `AccountID` as a string without FK constraint) to support some accounts in memory only sounds like a very bad idea to me.